text/entities/0264-resource-and-entities.md

```diff
@@ -0,0 +1,466 @@
+
+## Trade-offs and mitigations
+
+The design proposed here attempts to balance non-breaking (backwards and forwards compatible) changes with the need to improve problematic issues in the Specification.  Given the inability of most SDKs to implement the current Resource merge specification, breaking this should have little effect on actual users.  Instead, the proposed merge specification should allow implementations to match current behavior and expectation, while evolving for users who engage with the new model.
+
+## Prior art and alternatives
+
+Previously, we have a few unaccepted oteps, e.g. ([OTEP 208](https://github.com/open-telemetry/oteps/pull/208)).  Additionally, there are some alternatives that were considered in the Entities WG and rejected.
+
+Below is a brief discussion of some design decisions:
+
+- **Only associating one entity with a Resource.**  This was rejected, as too high a friction point in evolving semantic conventions and allowing independent systems to coordinate identity + entities within the OpenTelemetry ecosystem.  Eventually, this would force OpenTelemetry to model all possibly entities in the world and understand their interaction or otherwise prevent non-OpenTelemetry instrumentation from interacting with OpenTelemetry entities.
+- **Embed fully Entity in Resource.** This was rejected because it makes it easy/trivial for Resource attributes and Entities to diverge.  This would prevent the backwards/forwards compatibility goals and also require all participating OTLP users to leverage entities. Entity should be an opt-in / additional feature that may or may not be engaged with, depending on user need.
+- **Re-using resource detectoin as-is** This was reject as not having a viable compatibility path forward.  Creating a new set of components that can preserve existing behavior while allowing users to adopt the new functionality means that users have better control of when they see / change system behavior, and adoption is more obvious across the ecosystem.
+
+## Future Posibilities
+
+This proposal opens the door for addressing issues where an Entity's lifetime does not match an SDK's lifetime, in addition to providing a data model where mutable (descriptive) attributes can be changed over the lifetime of a resource without affecting its idnetity.  We expect a follow-on OTEP which directly handles this issue.
+
+## Use Cases
+
+Below are a set of use cases to help motivate this design.
+
+### SDK - Multiple Detectors of the same Entity type
+
+Let's consider the interaction of the SDK in the presence of multiple registered
+entity detectors:
+
+```mermaid
+flowchart LR
+    SDK["`**SDK**`"] -->|OTLP| BACKEND["`**Backend**`"]
+    SDK -.- RC((Resource Coordinator))
+    RC -.- OTEL_DETECTOR((OpenTelemetry Default Resource Detection))
+    RC -.- GCP_DETECTOR((Google Cloud Specific Resource Detection))
+    GCP_DETECTOR -. Detects .-> GCE{{gcp.gce}}
+    GCP_DETECTOR -. Detects .-> GCPHOST{{"host (gcp)"}}
```

Comment:
nit: diagram is broken

Reply:
I'm not seeing the breakage on my view, but it certainly WAS broken before [f679af2](https://github.com/open-telemetry/oteps/pull/264/commits/f679af21d880947c715f1e701e90bfe930805d07).  Please reopen if it's still broken.